enhancements/sig-architecture/6-placements/README.md

```diff
@@ -416,26 +333,14 @@ Limitations:
   - The number of ManagedClusterSets might be very large;
   - The new added ManagedClusterSets cannot be bound to the namespace automatically;
 
-### Workflows
+### Workflow: Placement Reconciliation
 
-#### Workflow 1: Placement Reconciliation
-1. If the placement has old placementdecisions, fetch them all and get a slice of old matched ManagedClusters;
-2. Evaluate each cluster predicates (see Workflow 2 below for more details) and get the union of all predicates result;
-3. Enforce the affinity constrains on the union and remove ManagedClusters if necessary;
-4. If the placement has no old placementdicisions yet, create placementdicisions;
-5. Otherwise, update the existing placementdicisions, and add/remove placementdicisions if necessary.
-
-#### Workflow 2: Evaluate a cluster predicate
-1. If `ClusterSets` is not empty, get ManagedClusters belongs to intersection of `ClusterSets` and the ManagedClusterSets bound to the placement namespace;
-2. Otherwise get ManagedClusters from the ManagedClusterSets bound to the placement namespace;
-3. If `RequiredDuringSchedulingRequiredDuringExecution` is specified, filter ManagedClusters with label/claim selectors;
-4. If `NumberOfClusters` is not specified, return all matched ManagedClusters;
-5. Otherwise, get the intersection of old matched ManagedClusters and new matched ManagedClusters;
-6. If `NumberOfClusters` is equal to the size of the intersection, return ManagedClusters in the intersection;
-7. If `NumberOfClusters` is less than the size of the intersection, select random ManagedClusters from the intersection and return;
-8. Otherwise, return ManagedClusters in the intersection with some random additional ManagedClusters from the remaining of the new matched ManagedClusters;
-
-**Note:** The old matched ManagedClusters are preferred during the predicate evaluation, which makes the placement decisions as stable as possible.
+1. Get all clustersetbindings in the placement namespace;
+2. Get eligible clustersets for the placement. A clusterset is eligible for a placement if it is bound to the placement namespace, and it is the .spec.clustersets if it is not empty;
+3. Find out all available clusters for the placement;
+4. Filter clusters with cluster predicates;
+5. Select clusters and generate cluster decisions;
+6. Bind the cluster decisions into placementdecisions;
```

Comment:
step 6 is not clear. Saying  bind... is confusing. Maybe step 5 only is sufficient.

Reply:
Step 6 removed.